src/components/BestSeller.jsx

```diff
@@ -1,38 +1,67 @@
 import axios from "axios";
 import { useState, useEffect } from "react";
+
 function BestSeller() {
   const [catagories, setCatagories] = useState([]);
+
   useEffect(() => {
     axios
       .get("https://fakestoreapi.com/products")
       .then((response) => {
-        setCatagories(response.data.slice(0,16));
+        setCatagories(response.data.slice(0, 12));
       })
       .catch((err) => {
         console.log("there is an error", err);
       });
   }, []);
+  const CatagoryList = [
+    catagories.slice(0, 4),
+    catagories.slice(4, 8),
+    catagories.slice(8, 12),
+  ];
   return (
-    <div className="border border-red-400">
+    <div className=" p-6 w-[95%] mx-auto">
```

Comment:
how about max width

Reply:
what will happen if the screen size is too large. there is no limit in upper bound